CMakeLists.txt

```diff
@@ -52,6 +164,7 @@ target_link_libraries(
   ${IRODS_EXTERNALS_FULLPATH_BOOST}/lib/libboost_filesystem.so
   ${IRODS_EXTERNALS_FULLPATH_BOOST}/lib/libboost_system.so
   ${IRODS_EXTERNALS_FULLPATH_ARCHIVE}/lib/libarchive.so
+  ${IRODS_EXTERNALS_FULLPATH_CLANG_RUNTIME}/lib/libc++abi.so.1
```

Comment:
This was needed to help Cen7 find the SO at runtime. I'm guessing the link_libraries call on **line 144** is supposed to take care of this, but apparently that doesn't work on CentOS (although it does for Ubuntu).